ga4gh/datamodel/datasets.py

```diff
@@ -170,6 +209,14 @@ def __init__(self, localId, dataDir, backend):
                 variantSet = variants.HtslibVariantSet(
                     self, localId, relativePath, backend)
                 self.addVariantSet(variantSet)
+            # Variant annotations sets
```

Comment:
I think we should set up some linkage here between the parent variantSet and the VariantAnnotationSet. At the moment when we set a ``variantannotationsets-search`` command the ``variantSet`` field is null. To resolve this we need a reference to the variantSet. The easiest way to do this is to pass the ``variantSet`` object in here as a parameter to the constructor of the HtslibVariantAnnotationSet object.